CareKit/CareCard/OCKCareCardViewController.h

```diff
@@ -144,6 +145,11 @@ OCK_CLASS_AVAILABLE
  */
 @property (nonatomic, readonly, nullable) OCKCarePlanEvent *lastSelectedInterventionEvent;
 
+/** 
+ A reference to the `UITableView` contained in the view controller
+ */
+@property (nonatomic, nonnull) UITableView *tableView;
```

Comment:
I had it set to allow writing as well so that it would be possible to replace the default `tableViewHeader` with a different. I believe setting the tableview to readonly would prevent setting it's `tableViewHeader` property, wouldn't? 

If I'm mistaken about that, then readonly would be appropriate.

Another possible compromise between the two would be to add public getters and setters that jump over the tableView and give direct access to the table views' headers.